- Don't install libnetpbm.a and libnetpbm.so into a sep link dir
- Try to fix parallel building
- Make the libopt tool less 'helpful'
- Remove pointless echo during build

--- lib/Makefile
+++ lib/Makefile
@@ -249,7 +249,7 @@
 .PHONY: install.staticlib
 install.staticlib: $(PKGDIR)/link
 	$(INSTALL) -c -m $(INSTALL_PERM_LIBS) libnetpbm.$(STATICLIBSUFFIX) \
-	  $(PKGDIR)/link
+	  $(PKGDIR)/lib
 
 # Install a shared library stub -- the ".so" file used at link time to
 # prepare a program for dynamically linking a library at run time 
@@ -257,9 +257,9 @@
 install.sharedlibstub: $(PKGDIR)/link
 ifeq ($(NETPBMLIBTYPE),unixshared)
 # install the link-time (.so) links to the runtime libraries
-	cd $(PKGDIR)/link ; \
+	cd $(PKGDIR)/lib ; \
           rm -f libnetpbm.$(NETPBMLIBSUFFIX); \
-          $(SYMLINK) ../lib/libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
+          $(SYMLINK) libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
             libnetpbm.$(NETPBMLIBSUFFIX)
 endif
 ifeq ($(NETPBMLIBTYPE),dll)
--- GNUmakefile
+++ GNUmakefile
@@ -87,6 +87,9 @@
 
 .PHONY: nonmerge
 nonmerge: $(PRODUCT_SUBDIRS:%=%/all)
+
+$(PRODUCT_SUBDIRS:%=%/all): buildtools/all
+converter/all analyzer/all editor/all generator/all other/all: lib/all
 
 OMIT_CONFIG_RULE = 1
 include $(SRCDIR)/Makefile.common
--- buildtools/libopt.c
+++ buildtools/libopt.c
@@ -496,4 +496,5 @@
 main(int argc, char **argv) {
 
+    bool shut_the_hell_up = TRUE;
     bool error;
     bool runtime;  /* -runtime option has been seen */
@@ -508,4 +509,6 @@
         if (strcmp(argv[arg], "-runtime") == 0)
             runtime = TRUE;
+        else if (strcmp(argv[arg], "-v") == 0)
+            shut_the_hell_up = FALSE;
         else {
             if (strlen(argv[arg]) > 200)
@@ -534,4 +537,5 @@
     } else {
         fputs(outputLine, stdout);
+        if (shut_the_hell_up == FALSE)
         fprintf(stderr, "libopt results: '%s'\n", outputLine);
         retval = 0;
--- converter/other/jpeg2000/Makefile
+++ converter/other/jpeg2000/Makefile
@@ -57,7 +57,6 @@
 include $(SRCDIR)/Makefile.common
 
 $(BINARIES): %: %.o $(JASPERLIB_DEP) $(NETPBMLIB) $(LIBOPT)
-	echo deps: $^
 	$(LD) $(LDFLAGS) $(MATHLIB) $(LDLIBS) -o $@ $< \
 	 `$(LIBOPT) $(NETPBMLIB) $(JASPERLIB_USE)` $(JASPERDEPLIBS) -lm \
 	  $(RPATH) $(LADD)
